plugins/portal-history-ornaments.js

```diff
@@ -0,0 +1,299 @@
+// @author         Johtaja
+// @name           Portal History&Ornaments
+// @category       Highlighter
+// @version        0.2.0
+// @description    Display portal history as highlighters, ornaments
+
+
+// use own namespace for plugin
+var portalsHistory = {};
+window.plugin.portalHighlighterPortalsHistory = portalsHistory;
+
+portalsHistory.styles = {
```

Comment:
I see here a massive copy/paste, and it is not good.
If you need same functions as we already have in`highlight-portal-history.js` - use them directly.